lobsterpy/cohp/analyze.py

```diff
@@ -236,14 +236,21 @@ def __init__(self, chemenv, valences=None):
 
                     """
                     if valences is None:
-                        self.coordination_environments = [
-                            [{"ce_symbol": str(len(coord))}] for coord in chemenv
-                        ]
+                        self.coordination_environments = []
+                        for coord in chemenv:
+                            if coord != []:

```

Comment:
You should take the length instead. This is notnsafe.

Reply:
Fixed it